app/autotranslate/client/lib/autotranslate.js

```diff
@@ -61,7 +61,7 @@ export const AutoTranslate = {
 						if (!message.translations) {
 							message.translations = {};
 						}
-						if (subscription && subscription.autoTranslate !== message.autoTranslateShowInverse) {
+						if (!!(subscription && subscription.autoTranslate) !== !!message.autoTranslateShowInverse) {
```

Comment:
this is confusing, what is the motivation here?

if the code cannot be made clear, please add a comment describing what it is doing.